compiler/commands.nim

```diff
@@ -312,7 +312,7 @@ proc testCompileOption*(conf: ConfigRef; switch: string, info: TLineInfo): bool
   of "symbolfiles": result = conf.symbolFiles != disabledSf
   of "genscript": result = contains(conf.globalOptions, optGenScript)
   of "threads": result = contains(conf.globalOptions, optThreads)
-  of "taintmode": result = contains(conf.globalOptions, optTaintMode)
+  of "taintmode": result = false  # TODO: https://github.com/nim-lang/Nim/pull/15423/files/bc158f0b9a940044dfbf6701d50c0c3dbed2774e#r558482102
```

Comment:
that link points to nowhere because of how github works
```suggestion
  of "taintmode": result = false  # pending https://github.com/nim-lang/Nim/issues/16731
```